assets/pages/digital-subscription-landing/components/promotionPopUpReducer.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action, PromotionOptions } from './promotionPopUpActions';
+
+// ----- Setup ----- //
+
+export type FindOutMoreState = {
+  isPopUpOpen: boolean,
+  expandedOption: PromotionOptions,
+};
+
+const initialState: FindOutMoreState = {
+  isPopUpOpen: false,
+  expandedOption: 'Saturday',
+};
+
+function promotionPopUpReducer(
+  state: FindOutMoreState = initialState,
+  action: Action,
+) {
+  switch (action.type) {
+    case 'OPEN_POP_UP':
+      return Object.assign({}, state, {
+        isPopUpOpen: true,
+      });
```

Comment:
[Rest spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Rest_syntax_(parameters)#Spread_in_object_literals) is an option here if you want:

```jsx
return { ...state, isPopUpOpen: true };
```